.github/workflows/tests.yml

```diff
@@ -0,0 +1,55 @@
+name: tests
+
+on: [pull_request]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-version:
+          - "7.1"
```

Comment:
You'll need to handle the conflict with https://github.com/pepakriz/phpstan-exception-rules/pull/126 and drop the 7.1 support.